src/main/java/com/google/api/codegen/config/SampleConfig.java

```diff
@@ -50,14 +99,110 @@ public static Builder newBuilder() {
   @AutoValue.Builder
   public abstract static class Builder {
 
+    public abstract Builder id(String val);
+
+    public abstract Builder title(String val);
+
+    public abstract Builder description(String val);
+
+    public abstract Builder interfaceConfig(InterfaceConfig val);
+
+    public abstract Builder methodConfig(MethodConfig val);
+
+    public abstract Builder requestConfigs(ImmutableList<RequestFieldProto> val);
+
+    public abstract Builder responseConfigs(ImmutableList<ResponseStatementProto> val);
+
     public abstract Builder regionTag(String val);
 
     public abstract Builder callingForm(CallingForm val);
 
+    public abstract Builder callingPattern(String val);
+
     public abstract Builder valueSet(SampleValueSet val);
 
     public abstract Builder type(SampleSpec.SampleType val);
 
     public abstract SampleConfig build();
   }
+
+  public static ImmutableList<SampleConfig> create(
```

Comment:
Could you add a test for this method?

Reply:
It's kind of hard to add a unit test for this method, since we have to build a lot of fake `interfaceConfigs` and `methodConfigs`. Once we have the baselines in, as long as all the existing samples are still generated I think we should be good.

Reply:
oh, Java.....

Reply:
I dislike architectures that discourage us from writing unit tests by making it onerous. I think you're right about coverage for this function via the baselines, though, so we can make do with that.